front/app/modules/commercial/flag_inappropriate_content/admin/components/ActivityWarningsTab.tsx

```diff
@@ -17,7 +17,7 @@ interface Props {
   onData: (data: InsertConfigurationOptions<ITabItem>) => void;
 }
 
-const ActivityTab = ({
+const ActivityWarningsTab = ({
```

Comment:
I saw Luuc's comment in the other pr and I agree, it might be a good idea to turn this into a hook or even utility function, maybe?

Reply:
Will reply in the other PR!